src/hope_dedup_engine/apps/api/models/deduplication.py

```diff
@@ -1,12 +1,12 @@
-from typing import Any, override
+from typing import Any, Final, override
 from uuid import uuid4
 
 from django.conf import settings
 from django.db import models
 
 from hope_dedup_engine.apps.security.models import ExternalSystem
 
-REFERENCE_PK_LENGTH = 100
+REFERENCE_PK_LENGTH: Final[int] = 100
```

Comment:
is this still used? @vitali-yanushchyk-valor

Reply:
> is this still used? @vitali-yanushchyk-valor

Yes - it's other part of code. @domdinicola